docs/body.md

```diff
@@ -166,11 +203,11 @@ Daemons
 
 Changes
 -------
-
 3.0.4
 
 - Fix issue with locally defined monitoring templates after upgrade (ZPS-3817)
 - Fix handles bad proxy hostnames (ZPS-3819)
+- Adds `Case sensitive/Invert Expression/Regular Expressions` zPropperties into DataSource configuration (ZPS-3867)
```

Comment:
the code in PR #19 appears to have datasource properties, not ZProperties, and your regular expression propertiy only contains a single regular expression, not multiple regular expressions